youtube-channels.csv

```diff
@@ -469,3 +469,4 @@ UCz27K_YRbqL-hFW1G6-lbLA, Metroid Vibes, https://www.youtube.com/channel/UCz27K_
 UCjE9ZtzIAMX90bW-fqvtxlw, onlyraretracks, https://www.youtube.com/channel/UCjE9ZtzIAMX90bW-fqvtxlw
 UCSPWqz8IyDrfyDlYXCPBSnw, Ogurecsound, https://www.youtube.com/channel/UCSPWqz8IyDrfyDlYXCPBSnw
 UCMIZSbYWOQMPX81pqKV-Opg, Supermarkt, https://www.youtube.com/channel/UCMIZSbYWOQMPX81pqKV-Opg
+UC0pNFcRCfLmnmvhYqGP51jQ, Pierre B, https://www.youtube.com/c/PierreBII
```

Comment:
The ID here refers to a non-music channel, wrong copy pasta?

--------

And https://www.youtube.com/c/PierreBII is already present here:

https://github.com/mirrorfm/data/blob/272b47457c2cbb5c7614f91d556c337c26a4408b/youtube-channels.csv#L60
https://mirror.fm/youtube/UCsIzfsOcObtsEXOwtsFlcFA/pierre-b/
https://open.spotify.com/playlist/3FrnJ32FCq3bic2PSBF05P?si=a234590717e743cd